src/main/java/org/jahia/services/usermanager/ldap/LDAPUserGroupProvider.java

```diff
@@ -201,6 +205,38 @@ public JahiaGroup getGroup(String name) throws GroupNotFoundException {
         }
     }
 
+    public List<Member> getDynamicGroupMembers(String userId, String groupName) {
```

Comment:
Just for clarity, change the signature to something like `boolean isMemberOfDynamicGroup(userId, groupName)` . The method does not (always) return the list of all members of the group. Can also be set as private.